### DISCUSSION TITLE
Load Icons from ttf file

### DISCUSSION THREAD

**Discussion by ravihani** [wix/react-native-ui-lib#2398] [Q&A]
I could not find it anywhere in the documenation so was wondering is there a way to load Icons from the .ttf file into the loadAssetsGroup.   Instead of specifiying each image, is it possible to load the ttf file and use Icons?

Assets.loadAssetsGroup('icons', {
  icon1: require('icon1.png'),
  icon2: require('icon2.png'),
  icon3: require('icon3.png'),
});

**Comment by ethanshar** [comment]
As long RN Image component supports it as an image source you can load whatever you want.
